### ISSUE TITLE
Probably a bug in the queue management

### ISSUE THREAD

**Issue by certrik** [ail-project/ail-framework#84]
In `screen -r Queue_AIL` I can see 
```
DomClassifier has no publisher.
SentimentAnalysis has no publisher.
DumpValidOnion has no publisher.
Duplicates has no publisher.
ModuleStats has no publisher.
WebStats has no publisher.
Indexer has no publisher.
Languages has no publisher.
Traceback (most recent call last):
  File "./QueueIn.py", line 24, in <module>
    run(args.config_section)
  File "./QueueIn.py", line 12, in run
    p.populate_set_in()
  File "/data/AIL/ail-framework/bin/Helper.py", line 155, in populate_set_in
MISP_The_hive_feeder has no publisher.
    self.pubsub.setup_subscribe(src)
  File "/data/AIL/ail-framework/bin/Helper.py", line 58, in setup_subscribe
    addresses = self.config.get(conn_name, 'address')
  File "/usr/lib/python3.7/configparser.py", line 780, in get
    d = self._unify_values(section, vars)
  File "/usr/lib/python3.7/configparser.py", line 1146, in _unify_values
    raise NoSectionError(section) from None
configparser.NoSectionError: No section: 'ZMQ_JSON'
Release has no publisher.
submit_paste has no suscriber
Importer_Json input queue finished.
Duplicates output queue finished.
Indexer output queue finished.
DomClassifier output queue finished.
Languages output queue finished.
DumpValidOnion output queue finished.
WebStats output queue finished.
ModuleStats output queue finished.
MISP_The_hive_feeder output queue finished.
SentimentAnalysis output queue finished.
Release output queue finished.
submit_paste input queue finished.
```

**Comment by lucadigregorio** [comment]
Me too, i open another issue `Stuck Queues`, i think it's the same problem. @Terrtia or @mokaddem -> That's what i can also see from the Queue_AIL when queues starts to be stuck

**Comment by Terrtia** [comment]
Hi @certrik, @lucadigregorio !

Thanks for the report!

@lucadigregorio This error doesn't have any impact on the queue manager. It throw an error if a module doesn't have any subscribers.

Fixed with 9ae21171cca2e72203c7c654e3bce4c52e21e1da
